packages/react-components/src/store/config.ts

```diff
@@ -0,0 +1,12 @@
+import { StateCreator } from 'zustand/esm';
+import Cookies from 'js-cookie';
+
+export type Flags = 'useEcharts';
+export interface ConfigState {
+  config: Record<Flags, boolean>;
+}
+export const createConfigSlice: StateCreator<ConfigState> = () => ({
+  config: {
+    useEcharts: !!Cookies.get('USE_ECHARTS'),
```

Comment:
Could we use local storage instead of cookies? We have to get consent from users to use cookies.

Reply:
Fair call, updating to localStorage